charts/sourcegraph/tests/postgresAdditionalConfig_test.yaml

```diff
@@ -0,0 +1,36 @@
+---
+suite: postgresAdditionalConfig_01
+templates:
+- pgsql/pgsql.ConfigMap.yaml
+- codeintel-db/codeintel-db.ConfigMap.yaml
+- codeinsights-db/codeinsights-db.ConfigMap.yaml
+tests:
+- it: should have additional config rendered when *.additionalConfig is defined
+  set:
+    pgsql:
+      additionalConfig: |
+        max_connections = 1000000000000000000
+    codeIntelDB:
+      additionalConfig: |
+        max_connections = 1000000000000000000
```

Comment:
I like to use different values for each test to avoid false positives - right now, the test would pass if the template were accidentally always referring to `pgsql.additionalConfig` instead of their own references.